js/data/bucket/fill_bucket.js

```diff
@@ -15,7 +16,7 @@ const fillInterfaces = {
             components: 2,
             type: 'Int16'
         }]),
-        elementArrayType: new ElementArrayType(1),
+        elementArrayType: new ElementArrayType(3),
```

Comment:
It worked with `1` before because `FillBucket#addFeature` [emplaced one index at a time](https://github.com/mapbox/mapbox-gl-js/pull/3461/files#diff-3d556b8c468e9f1285f261f235775d71L96). Now it emplaces triangles, for consistency with `FillExtrudeBucket` (and native).